ProcessMaker/Providers/ProcessMakerServiceProvider.php

```diff
@@ -422,8 +420,6 @@ public static function setPackageBootStart(string $package, $time): void
      */
     public static function setPackageBootedTime(string $package, $time): void
     {
-        $package = ucfirst(\Str::camel(str_replace(['ProcessMaker\Packages\\', '\\'], '', $package)));
-
         self::$packageBootTiming[$package]['end'] = $time;
```

Comment:
validate if  self::$packageBootTiming is defined

Reply:
a validation was added